Example/Views/JS/Author.cshtml

```diff
@@ -9,6 +9,7 @@
 <script>
     document.getElementById('code').innerHTML = library.json.prettyPrint(@Html.Raw(ViewBag.JSON));
 </script>
+<script src="//authorapi.learnosity.com?v0.10"></script>
 <script>
     var initOptions = @Html.Raw(ViewBag.JSON);
     LearnosityAuthor.init(initOptions);
```

Comment:
@gonzalozawa and in here please do not call `LearnosityAuthor.init(initOptions);`